doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,444 @@
+
+### Architecture
+<details>
+ <summary>The installer target architecture</summary>
+
+ #### Required Field
+
+ This key represents the hardware architecture targeted by the installer. The Windows Package Manager will attempt to determine the best architecture to use. If emulation is available and the native hardware architecture does not have a supported installer, the emulated architecture may be used.
+ </details>
+
+### InstallerUrl
+<details>
+ <summary>The installer Url</summary>
+
+ #### Required Field
+
+ This key represents the URL to download the installer.
+
+</details>
+
+### InstallerSha256
+<details>
+ <summary>Sha256 is required. Sha256 of the installer</summary>
+
+ #### Required Field
+
+ This key represents the SHA 256 hash for the installer. It is used to confirm the installer has not been modified. The Windows Package Manager will compare the hash in the manifest with the calculated hash of the installer after it has been downloaded.
+
+ >Note:  The [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) can be used to determine the SHA 256 of the installer. The `winget hash &lt;pathToInstaller&gt;` command can also be used to determine the SHA 256 of the installer.
+
+</details>
+
+### SignatureSha256
+<details>
+ <summary>SignatureSha256 is recommended for appx or msix. It is the sha256 of signature file inside appx or msix. Could be used during streaming install if applicable</summary>
+
+ #### Optional Field
+
+ This key represents the signature file inside an MSIX installer. It is used to provide streaming install for MSIX packages.
+
+ >Note: MSIX installers must be signed to be included in the Microsoft community package repository. If the installer is an MSIX this signature should be included in the manifest. The [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) can be used to determine the signature SHA 256. The `winget hash &lt;pathToInstaller&gt; --msix` command can also be used to determine the signature SHA 256.
+
+</details>
+
+### Installers
+<details>
+ <summary>Array of package installers</summary>
+
+ #### Required Field
+
+ This key must be present for each installer for this version of the package. There may be multiple installer nodes to support different architectures, locales, install scopes (User vs. Machine)
+</details>
+
+### ManifestType
+<details>
+ <summary>The manifest type</summary>
+
+ #### Required Field
+
+ This key must have the value "version". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.
+</details>
+
+### ManifestVersion
+<details>
+ <summary>The manifest syntax version</summary>
+
+ #### Required Field
+
+ This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.
+</details>
```

Comment:
Do you think we should provide sample manifests (or links to them if they already exist) so the user can reference?

Reply:
I'm leaning away from doing that as they tend to change and might get deleted.